extensions/amp-story/1.0/_locales/en.js

```diff
@@ -98,6 +98,11 @@ const strings = {
     string: 'Play video',
     description: 'Label for a button to play the video visible on the page.',
   },
+  [LocalizedStringId.AMP_STORY_HAS_NEW_PAGE_TEXT]: {
+    string: 'Story updated',
+    description:
+      'Label indicating that the story has been updated with (a) new page(s).',
```

Comment:
Since the translators have 0 context about the product, should we make this description less specific, and maybe say that new content is available?

Reply:
Maybe let's try "Label that indicates that additional content has been added to a story"

WDYT?